### PR TITLE
 config: improve the way configs are done

### PR DESCRIPTION
**Pull request by grvcoelho**
## Description

Move bradesco provider configuration from credstash to a static config
filed. This way, we move from an async approach to a sync one.

Also, remove everything related to lambda invocation.

This closes https://github.com/pagarme/ghostbusters/issues/13 (internal)

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](../CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation
1. I feel comfortable with the code I wrote, and I'm not ashamed to show it to my friends

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

**Comment by otaviopace** [comment]
Não acha válido remover do arquivo `src/resources/boleto/index.js` as funções `register`, `processBoletosToRegister` e `configureContext`, além de seus testes no commit `lambda: remove old lambda invocation library`?

File: src/providers/bradesco/index.js [hunk 1]
```diff
@@ -14,54 +14,53 @@ const { format } = require('./formatter')
...
 const { makeFromLogger } = require('../../lib/logger')
-const { getCredentials } = require('../../lib/credentials')
 const responseCodeMap = require('./response-codes')
 
-const { endpoint } = prop('bradesco', getConfig())
+const {
+  api_key: apiKey,
+  endpoint,
+  merchant_id: merchantId,
+} = prop('bradesco', getConfig())
```
Comment (otaviopace) [inline src/providers/bradesco/index.js]: acha justificável a utilização de `prop` aqui?

File: src/providers/bradesco/index.js [hunk 2]
```diff
@@ -14,54 +14,53 @@ const { format } = require('./formatter')
...
-    .spread((merchantId, securityKey) => {
-      const authorization = encodeBase64(`${merchantId}:${securityKey}`)
+const buildHeaders = () => {
+  const authorization = encodeBase64(`${merchantId}:${apiKey}`)
 
-      return {
-        Authorization: `Basic ${authorization}`,
-      }
-    })
+  return {
```
Comment (otaviopace) [inline src/providers/bradesco/index.js]: gostei que ficou sincrono agora :)

File: src/providers/bradesco/index.js (hunk 1 above)
Comment (grvcoelho) [reply src/providers/bradesco/index.js]: Só deixei do jeito que estava, não sei se vale mexer nisso.

**Comment by grvcoelho** [comment]
> Não acha válido remover do arquivo src/resources/boleto/index.js as funções register, processBoletosToRegister e configureContext, além de seus testes no commit lambda: remove old lambda invocation library?

Boa! Removi as funções `register` e `configureContext`. A função `processBoletosToRegister` você vai tirar no PR #274 ,né?

**Comment by otaviopace** [comment]
No caso o PR #274 já foi mergeado,

**Pull request merged**